flocker/testtools/_base.py

```diff
@@ -77,6 +82,8 @@ class AsyncTestCase(testtools.TestCase):
     Base class for asynchronous test cases.
```

Comment:
Absolutely. Likewise the `mktemp` mixin. The [plan](https://clusterhq.atlassian.net/browse/FLOC-3281) is to move synchronous test case over to testtools once we've got a regression-free `AsyncTestCase`. We are very, very close to this.

Added a comment here along these lines.